index.html

```diff
@@ -30,18 +32,44 @@
             <a class=menu_item>Contact</a>
             <img class="menu_mail" src="./img/mail.png" alt="Email Icon">
         </nav>
-        <nav class="nav-toggle" id="navToggle">
+        <nav id="navToggle" class="nav-toggle">
             <img src="./img/menu.png" alt="Hamburger Menu">
+
+            <div class="Hamburger-me">
+                <img class="close" id="cancel" src="./img/Cancel.png" alt="Close">
```

Comment:
- [ ] Your _close button_ is **overlapping** with the menu items. Kindly improve upon this aspect by ensuring that the _close button_ is positioned at the same place as the _hamburger menu_, and not cluttered with the mobile menu items

**Your mobile menu overlay (when the hamburger menu is clicked):**
![image](https://user-images.githubusercontent.com/12745474/132320303-9f439705-500e-40ce-9e88-efbc0f64d5da.png)